source/guides/recipes/autoformatting.md

```diff
@@ -0,0 +1,55 @@
+(autoformatting)=
+# Autoformatting
+
+[`nixfmt`](https://github.com/NixOS/nixfmt) is the official Nix autoformatter.
+It is not yet wired up to any official tooling. Subscribe to
```

Comment:
Very minor, but the wording feels backwards to me... The onus should be on the tooling to wire up; it is out of nixfmt's hands what tooling decides to use it.

I'm also struggling to phrase it, but maybe:

```suggestion
Official tooling has yet to be integrated with nixfmt. Subscribe to
```

or:

```suggestion
Official tooling currently doesn't use nixfmt out of the box. Subscribe to
```